libraries/utils/glfw_utils/glfw_window.cpp

```diff
@@ -0,0 +1,152 @@
+#include "glfw_window.hpp"
+
+#include <bnb/effect_player/utility.hpp>
+#include <bnb/utils/defs.hpp>
+
+#include <glad.h>
+
+namespace bnb::gl
+{
+
+    /* glfw_window::glfw_window */
```

Comment:
I think such comments are excessive, please reconsider their usage - I'd prefer to remove them to make the style closer to SDK.